Fix outdated riemann_client dependency

## Bug description

Riemann exporter now relies on riemann_client 0.9, which is flagged as outdated and should be replaced.

The riemann_client project seems inactive, we should consider

## To Reproduce

Compile scaphandre with cargo build, see the error : 

> warning: the following packages contain code that will be rejected by a future version of Rust: riemann_client v0.9.0
> note: to see what the problems were, use the option `--future-incompat-report`, or run `cargo report future-incompatibilities --id 1`

## Expected behavior

No warning, use an up to date dependency.

## Additional context

Pinging @uggla if available as he knows the exporter very well.

To make the mention in NixOS/nixpkgs explicit: with a recent rust toolchain, scaphandre FTBFS. 
```
  Compiling riemann_client v0.9.0
error[E0658]: custom inner attributes are unstable
 --> /home/jonesru/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/riemann_client-0.9.0/src/proto/mod_pb.rs:9:4
  |
9 | #![rustfmt::skip]
  |    ^^^^^^^^^^^^^
  |
  = note: see issue #54726 <https://github.com/rust-lang/rust/issues/54726> for more information
```
Mentions of this issue seem to say reimann_client is too old... but the error is reporting that the feature used is too new ("unstable"). I'm aware rust compiles source with old versions as specified by that source, but if this was unstable in an older version, how did it ever work at all with a stable toolchain? (If it didn't, the build instructions need updated to mention a nightly toolchain is needed, ideally stating it explicitly rather than giving a version number that won't be as widely meaningful.)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,9 +16,8 @@ loggerv = "0.7"
 log = "0.4"
 clap = { version = "4.2", features = ["cargo", "derive"] }
 regex = "1.7"
-riemann_client = { version = "0.9.0", optional = true }
+riemann_client = { version = "1.0.0", optional = true }
 hostname = "0.3.1"
-protobuf = "2.28.0"
 serde = { version = "1.0", features = ["derive"], optional = true }
 serde_json = { version = "1.0", optional = true }
 ordered-float = "2.0"

--- a/src/exporters/riemann.rs
+++ b/src/exporters/riemann.rs
@@ -49,9 +49,9 @@ impl RiemannClient {
         event.set_host(metric.hostname.to_string());
         event.set_service(metric.name.to_string());
         event.set_state(metric.state.to_string());
-        event.set_tags(protobuf::RepeatedField::from_vec(metric.tags.clone()));
+        event.tags = metric.tags.clone();
         if !attributes.is_empty() {
-            event.set_attributes(protobuf::RepeatedField::from_vec(attributes));
+            event.attributes = attributes;
         }
         event.set_description(metric.description.to_string());
 
@@ -120,11 +120,11 @@ pub struct ExporterArgs {
     )]
     pub mtls: bool,
 
-    /// CA certificate file (.pem format)
+    /// CA certificate file (.crt format)
     #[arg(long = "ca", requires = "mtls")]
     pub ca_file: Option<String>,
 
-    /// Client certificate file (.pem format)
+    /// Client certificate file (.crt format)
     #[arg(long = "cert", requires = "mtls")]
     pub cert_file: Option<String>,
 

